test/ajv-validator.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as sinon from 'sinon';
+import Ajv from 'ajv';
+import { expect } from 'chai';
+import { AjvValidator } from '../lib';
+
+describe('AjvValidator', () => {
+  let ajvValidator: AjvValidator;
+
+  beforeEach(() => {
+    ajvValidator = new AjvValidator();
```

Comment:
Я смотрю ниже много где моканье прототипа, а это не самый лучший подход, потому что ты мокаешь глобальный прототип, а мы тестируем только `Ajv` который в классе `AjvValidator`. Я предлагаю тебе сделать псевда глобальный мок для удобства

```suggestion
  let ajvValidator: AjvValidator;
  let mockAjv: sinon.SinonMock;

  beforeEach(() => {
    ajvValidator = new AjvValidator();
    mockAjv = sinon.mock((ajvValidator as any)._ajv);
```

и дальше в каждом кейсе делать примерно так
```ts
it('should be success result if schema not exist', async () => {
    mockAjv.expects('getSchema').returns(false);
    mockAjv.expects('getSchema').once().called();
    const result = await ajvValidator.validateById('', {});
    expect(result.success).is.true;
    mockAjv.verify();
  });
```
Только опять же таки чекни синтаксис, пишу все по памяти